python/CHANGELOG.md

```diff
@@ -7,3 +7,7 @@
 ## [1.0.0] - 2018-02-23
 ### Added
 - Initial implementation.
+
+## [1.1.0] - 2018-07-06
```

Comment:
Wrong section order. New entries are added on the top. See [this](https://github.com/techmo-pl/sarmata-client/blob/master/python/CHANGELOG.md), for example. Correct.